libclamav/others.h

```diff
@@ -320,6 +321,7 @@ struct cl_engine {
     uint32_t ac_mindepth;
     uint32_t ac_maxdepth;
     char *tmpdir;
+    char* certs_directory;
```

Comment:
Check if I'm actually using this. I can't remember... i think it's just in the engine and not used in the ctx.